README.md

```diff
@@ -111,6 +111,23 @@ done with care.
 Then, install Rust using [rustup](https://rustup.rs/). Install Rust
 toolchains for cross-compilation by executing `scripts/install-toolchains.sh`.
 
+After that, add `$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/HOST/bin` to your $PATH,
+where HOST is a descriptor for your host cpu architecture (e.g. linux-x86_64) and likely
+the name of the only subdirectory within `$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt`.
+
+Then also add the correct clang-linkers to `~/.cargo/config.toml` (create it if it doesn't exist):
+
+```
+[target.armv7-linux-androideabi]
+linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/armv7a-linux-androideabi18-clang"
```

Comment:
hi, thanks for contributing but I think this is automatically done in `ndk-make.sh` via `CARGO_TARGET_ARMV7_LINUX_ANDROIDEABI_LINKER`, etc environment variables instead of writing it in `~/.cargo/config.toml`

see `ndk-make.sh` script line 48.

Reply:
i also had a closer look, indeed, these steps are no longer needed since this refactoring: https://github.com/deltachat/deltachat-android/commit/18a19f98b6acd9180edc7a50aef8ca197634d292

before, more or less the suggestion of this pr was present in the comment / readme.